lib/travis/listener/app.rb

```diff
@@ -182,7 +185,7 @@ def event_details
 
       def decoded_payload
         @decoded_payload ||= begin
-          schema = 
+          schema =
             case event_type
             when 'push'
               Schemas::PUSH
```

Comment:
One thing to note is we're using the stripped down `FALLBACK` schema which I don't believe is enough, so I need more info/research into what exactly `github-sync` needs for processing this event. Any additional info is welcome~